demos/dotnet/dotnet/project.json

```diff
@@ -0,0 +1,12 @@
+﻿{
+  "dependencies": {
```

Comment:
dotnet.dotnetproj contains all the required packages to self-host (along with the explicit references.txt and dependencies.txt files) using nugget as of now. The project.json file is used by dnu to restore the packages so that dotnet can be compiled into a coreclr app using msbuild and the dotnet-core.csproj file.